docs/version0.37.md

```diff
@@ -75,6 +76,26 @@ A new message, `JVMPORT049I`, is added to help find the system dump files for `k
 
 For more information about system dumps and piped system dumps, see [System dumps on Linux](xdump.md#system-dumps-on-linux).
 
+### ![Start of content that applies to Java 19 plus](cr/java19plus.png) New JDK 19 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 424](https://openjdk.java.net/jeps/424): Foreign Function & Memory API (Preview)
+- [JEP 425](https://openjdk.java.net/jeps/425): Virtual Threads (Preview)
+- [JEP 426](https://openjdk.java.net/jeps/426): Vector API (Fourth Incubator)
+- [JEP 428](https://openjdk.java.net/jeps/428): Structured Concurrency (Incubator)
```

Comment:
I don't think this should be under the heading "The following features are supported by OpenJ9". Since it's not implemented, it's not supported. It will only apply to OpenJ9 once it's implemented.

Reply:
I can delete JEP 428 information then?

Reply:
I think we should list it under a different heading.